src/Altinn.Platform/Altinn.Platform.PDF/src/main/java/altinn/platform/pdf/models/FormLayoutElement.java

```diff
@@ -30,4 +34,18 @@ public String getType() {
   public TextResourceBindings getTextResourceBindings() { return textResourceBindings; }
 
   public void setTextResourceBindings(TextResourceBindings textResourceBindings) { this.textResourceBindings = textResourceBindings; }
+
+  public List<String> getChildren() { return children; }
+
+  public void setChildren(List<String> children) { this.children = children; }
+
+  public int getCount() { return count; }
+
+  public void setCount(int count) { this.count = count; }
+
+  public int getMaxCount() { return maxCount; }
+
+  public void setMaxCount(int maxCount) { this.maxCount = maxCount; }
+
```

Comment:
Tips: Med Lombok kan man få getter/setter++ med annotasjon slik at du slipper å skrive disse selv.
https://projectlombok.org/

Reply:
Ah, nice. IntelliJ har egentlig gjort denne jobben for meg med `alt+insert` + generate getters/setters.